src/sentry/models/organizationmapping.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import BoundedBigIntegerField, Model
+from sentry.db.models.base import control_silo_only_model
+
+
+@control_silo_only_model
+class OrganizationMapping(Model):
+    """
+    This model is used to:
+    * Map org slugs to a specific organization and region
+    * Safely reserve organization slugs via an eventually consistent cross silo workflow
+    """
+
+    __include_in_export__ = True
+
+    organization_id = BoundedBigIntegerField(db_index=True)
+    slug = models.SlugField(unique=True)
+    created = models.DateTimeField(default=timezone.now)
+    stripe_id = models.CharField(max_length=255, db_index=True)
```

Comment:
Note: As written, self hosted accounts end up with a bunch of empty strings in this column?  Is that going to break any validations or logic that may expect nulls?

Reply:
I'll make it explicitly nullable just to clarify the semantics.

Reply:
Thank you!